src/schemas/application.ts

```diff
@@ -25,6 +25,9 @@ export const basicsSaveSchema = applicationSaveSchema.pick({
   age: true,
 });
 
+export const personaSaveSchema = applicationSaveSchema.pick({
+  avatar: true,
```

Comment:
lol joy fix this, u didn't close the { or the (

Reply:
oop lost it in resolving merge conflict